src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/CustomSelection.cs

```diff
@@ -19,6 +21,7 @@ public class CustomSelectionNodeViewCustomization : DropDownNodeViewCustomizatio
         /// <param name="nodeView"></param>
         public void CustomizeView(CustomSelection model, NodeView nodeView)
         {
+            const double leftMargin = 40;
```

Comment:
hmm where does this 40 come from?

Reply:
It was already hardcoded I just move it to a const.

![image](https://github.com/DynamoDS/Dynamo/assets/61755417/be2406dd-f114-4d05-b0a1-6edb9bc3ebd3)

https://github.com/DynamoDS/Dynamo/blob/bb72d492d475b406bff9c50e46bac0f4ce050de9/src/Libraries/CoreNodeModelsWpf/NodeViewCustomizations/CustomSelection.cs#L34